src/mgr/ActivePyModules.h

```diff
@@ -27,6 +29,11 @@
 #include "DaemonState.h"
 #include "ClusterState.h"
 
+#define dout_context g_ceph_context
```

Comment:
Defining dout macros in a header risks contaminating any .cc files that directly or indirectly include this header -- in general code that requires logging statements belongs in a .cc file so